readme.md

```diff
@@ -38,6 +38,10 @@ You must define `arc` directive on element which you want to be focusable that a
 - `select`
 - `textarea`
 
+##### arc-exclude-this
```

Comment:
arc-exclude-this was added specifically to exclude _this_ element, a different one should be added to exclude all children. Or there could be a generic `arc-exclude` that can be used like `arc-exclude="this"` or `arc-exclude="tree"`